maths/shreedharacharya.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:math';
+//this function return a list of roots of a quadratic equation
+// [x1, x2] where x1 and x2 are roots of
+// aX^2 + bX + c = 0
+List shreedharacharya(double a,double b,double c) 
+{
+  double d = b*b - 4*a*c;
+  List A=[0,0];
+  if(d<0)
+  {
+    print('Imaginary roots');
+  }
+  else{
+    A[0] = (-b + pow(d,0.5))/(2*a);
```

Comment:
After switching to empty list you can just
```suggestion
    A.add((-b + pow(d,0.5))/(2*a));
```
You could also replace `pow(d,0.5)` with `sqrt(d)`

```suggestion
   A.add((-b + sqrt(d))/(2*a));
```